token-metadata/program/ProgrammableNFTGuide.md

```diff
@@ -1,8 +1,13 @@
 # Programmable NFT Guide
 
-Developer packages:
-* :crab: Rust crate: [v1.8.0](https://crates.io/crates/mpl-token-metadata/1.8.0)
-* :package: NPM package: [v2.8.0](https://www.npmjs.com/package/@metaplex-foundation/mpl-token-metadata/v/2.8.0)
+## Developer packages
+Token metadata
+* :crab: Rust crate: [v1.8.1](https://crates.io/crates/mpl-token-metadata/1.8.1)
+* :package: NPM package: [v2.8.1](https://www.npmjs.com/package/@metaplex-foundation/mpl-token-metadata/v/2.8.1)
+
+Auth rules
```

Comment:
Should we use the "full name"? E.g., Token Authorization Rules.

Reply:
Done